plugins/admin/cmd/kn-admin.go

```diff
@@ -17,12 +17,13 @@ import (
 	"fmt"
 	"os"
 
+	_ "k8s.io/client-go/plugin/pkg/client/auth/oidc"
 	"knative.dev/client-contrib/plugins/admin/core"
 )
 
 func main() {
 	if err := core.NewAdminCommand().Execute(); err != nil {
-		fmt.Println("failed to execute admin command:", err)
+		fmt.Println(err.Error())
```

Comment:
I would rather see a well defined string instead of the error. Because the content of `err.Error()` might be not user friendly